pulp_rpm/app/serializers/modulemd.py

```diff
@@ -29,6 +29,9 @@ class ModulemdSerializer(SingleArtifactContentUploadSerializer, ContentChecksumS
     version = serializers.CharField(
         help_text=_("Modulemd version."),
     )
+    static_context = serializers.BooleanField(
+        help_text=_("Modulemd static-context?"),
```

Comment:
It's arguably better to have something more explicit than a question mark for user facing descriptions.
Maybe `Modulemd static-context flag` ? or `Modulemd context semantics`? No good ideas come to mind :/

Reply:
Well, I think all the help-text needs some help. My reading here is, "if you don't know what these are already, you probably shouldn't be looking at this object" :) I thought of a few alternatives but they're all 'wordy', and stuck out in comparison. I do think the type-hint of '?' isn't useful - just realized it looks like we don't know what this field is, not like a type indicator. Will def change that.

Reply:
Updated the help-text a lot